pages/callback.tsx

```diff
@@ -23,7 +23,7 @@ export default function Connection() {
         redirectUri: home,
       })
     );
-  }, []);
+  });
```

Comment:
This will execute on every rerender, which is for sure not what we want.

Either do as:
```
useEffect(() => {
   ...
   // eslint-disable-next-line react-hooks/exhaustive-deps
}, []) 
```
or actually put all of the dependencies that it's complaining about - a perf hit for checking deps is usually negligible (shallow compare).